.gitignore

```diff
@@ -1,3 +1,4 @@
-node_modules/
-dist/

```

Comment:
Why was `dist/` removed?

Reply:
i added the .gitignore to the dist directory, so when checking the code out en running the code without error of missing dist folder